extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AbstractRequestWrapper.java

```diff
@@ -31,10 +31,10 @@
 public abstract class AbstractRequestWrapper
         implements HttpServerRequest, HttpServerRequestInternal {
 
-    protected final HttpServerRequestInternal delegate;
+    protected final HttpServerRequest delegate;
 
     protected AbstractRequestWrapper(HttpServerRequest request) {
```

Comment:
Dumb question, but wouldn't it be easier to change this constructor to accept a `HttpServerRequestInternal` instead?

Reply:
If that happens it means that other code paths should perform the cast instead (because vertx `internal` types are not exposed).